src/components/IndividualPageContent/ContentRow.tsx

```diff
@@ -22,7 +22,7 @@ export default function ContentRow({
         container
         rowSpacing={1}
         columnSpacing={4}
-        alignItems="center"
+        alignItems="start"
```

Comment:
I would make it as "center" because the values are not always a line of text. Sometimes it's a hash button, sometimes it's a json card. I feel it looks better when the label and value are center aligned with each other.

Reply:
The problem with vertically centering here is if there is a large content area to the right, the label could potentially be pushed below the viewport. 

In terms of hierarchy the label / title should be presented before (or at the same time) as it's content.